test/basics/demo-complex.html

```diff
@@ -0,0 +1,18 @@
+<script type='text/stache' can-autorender id='demo-html'>
+	{{#each items}}
+		<li>{{name}}</li>
+	{{/each}}
+</script>
+</script>
+
+<script src="../../node_modules/steal/steal.js" main="@empty" id='demo-source'>
+import "can-view-autorender";
+import "can-stache-bindings";
```

Comment:
I was wondering the same thing, but I don't know much about CanJS. Can you recommend a better "complex" test to do? I like this one because it takes a second (literally) to load.